Makefile

```diff
@@ -13,6 +13,8 @@ RUST_PROTOBUF_VERSION ?= 2.0.5
 RUST_VERSION ?= 1.29.0
 SWIFT_VERSION ?= 4.2
 UPX_VERSION ?= 3.95
+#only defined, will be used in future
+PROTOC_GEN_GOVALIDATORS_VERSION ?= 0.1
```

Comment:
Just use the latest commit (https://github.com/mwitkow/go-proto-validators/commit/0950a79900071e9f3f5979b78078c599376422fd) here

Reply:
Also, please sort these in alphabetical order